.github/workflows/scorecard.yml

```diff
@@ -0,0 +1,78 @@
+# This workflow uses actions that are not certified by GitHub. They are provided
+# by a third-party and are governed by separate terms of service, privacy
+# policy, and support documentation.
+
```

Comment:
Do you think it makes sense to remove this discalimer? It probably does not add much value.